Library/CreateBackingInput+Constructor.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import KsApi
+
+extension CreateBackingInput {
+  internal static func input(
+    from project: Project,
+    reward: Reward,
+    pledgeAmount: Double,
+    selectedShippingRule: ShippingRule?,
+    refTag: RefTag?,
+    paymentSourceId: String
+  ) -> CreateBackingInput {
+    let pledgeAmountDecimal = Decimal(pledgeAmount)
```

Comment:
A lot of this is duplicated from `CreateApplePayBackingInput` - I decided not to refactor out the shared parts because I know we're planning to consolidate the two mutations in the near future. When we do, we can simply merge the differences from `CreateApplePayInput` into this input.